test/interpolating/optional-paths.test.js

```diff
@@ -0,0 +1,23 @@
+var test  = require('tape')
+  , rhumb = require('../../src/rhumb')
+
+test('Interpolating should handle a path with an empty fixed part', function (t) {
+  t.plan(1)
+
+  t.equal(rhumb.interpolate('(/foo)', {}), '/foo'
+    , 'includes optional path when being interpolated')
+})
+
+test('Interpolating should handle a path with empty optional part', function (t) {
+  t.plan(1)
+
+  t.equal(rhumb.interpolate('/wibble()', {}), '/wibble'
+    , 'ignores empty optional path when being interpolated')
+})
```

Comment:
As stated below, I have changed my mind regarding error checking.  I think that the benefits of error  checking do outweigh the costs.